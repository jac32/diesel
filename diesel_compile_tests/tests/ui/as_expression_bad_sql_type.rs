#[macro_use]
extern crate diesel;

#[derive(AsExpression)]
#[sql_type(Foo)]
#[sql_type]
#[sql_type = "@%&&*"]
#[sql_type = "1omg"]
struct Lol;

fn main() {}
