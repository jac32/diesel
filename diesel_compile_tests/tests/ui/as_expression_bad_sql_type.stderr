error: `sql_type` must be in the form `sql_type = "value"`
 --> <macro expansion>:1:1
  |
1 | #[sql_type(Foo)]
  | ^

error: `sql_type` must be in the form `sql_type = "value"`
 --> <macro expansion>:1:1
  |
1 | #[sql_type]
  | ^

error: Invalid Rust type
 --> <macro expansion>:1:1
  |
1 | #[sql_type = "@%&&*"]
  | ^

error: Invalid Rust type
 --> <macro expansion>:1:1
  |
1 | #[sql_type = "1omg"]
  | ^

error: aborting due to 4 previous errors

